alpha/lib/model/LiveEntryServerNodeRecordedProperties.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @package Core
+ * @subpackage model
+ */
+class LiveEntryServerNodeRecordedProperties extends BaseObject
+{
```

Comment:
I think that this object should save entryId, serverNodeId & duration will be mush more clear.

And the serverNodeId may change during live stream so you may want to track which machine recorded which entry